run_unittests.py

```diff
@@ -1493,6 +1493,36 @@ def test_run_installed(self):
         env['LD_LIBRARY_PATH'] = installed_libdir
         self.assertEqual(subprocess.call(installed_exe, env=env), 0)
 
+    def test_components(self):
+        testdir = os.path.join(self.common_test_dir, '139 components')
+        test_data = [
+            (['-Dhello1=disabled'], 'disabled', 'enabled', 'disabled', 'enabled', 'enabled'),
+            (['-Dhello1=disabled', '-Dhello2=disabled'],
+              'disabled', 'disabled', 'disabled', 'enabled', 'enabled'),
```

Comment:
__[flake8]__
* [E127] continuation line over-indented for visual indent

<!-- Comment by [SideCI](https://sideci.com) -->